automatedteller.js

```diff
@@ -0,0 +1,83 @@
+$(document).ready(function(){
+  console.log("Your Mom");
+//
+// // change body to red on mouseover:
+//
+// $("body").on("mouseover", bodyRed);
+//
+//   function bodyRed() {
+//
+//     $("body").css("background", "red");
+//   }
+//
+// // log hello in console on click of checking deposit button:
+//
+// $("#checking .deposit").on("click", hello);
+//
+//   function hello()  {
+//     console.log("Hello.");
+//   }
+//
+// });
+
+$("#checking .deposit").on("click", function(e)  {
+    e.preventDefault();
+    var userDeposit = $("#checking .input").val();
+    var balanceAmount = $("#checking .balance").html();
+    var nUserDeposit = parseFloat(userDeposit);
+    var nBalanceAmount = parseFloat(balanceAmount.split("$")[1]);
+
+    console.log(nUserDeposit);
+    console.log(nBalanceAmount);
+
+    $("#checking .balance").html("$" + (nUserDeposit + nBalanceAmount).toFixed(2));
+
+});
+
+
+$("#checking .withdraw").on("click", function(e)  {
+  e.preventDefault();
+  var userWithdraw = $("#checking .input").val();
+  var balanceAmount = $("#checking .balance").html();
+  var nUserWithdraw = parseFloat(userWithdraw);
+  var nBalanceAmount = parseFloat(balanceAmount.split("$")[1]);
+
+  console.log(nUserWithdraw);
+  console.log(nBalanceAmount);
+
+  $("#checking .balance").html("$" + (nBalanceAmount - nUserWithdraw).toFixed(2));
+
```

Comment:
I would have included a conditional statement here to make sure this only happens if (nBalanceAmount-nUserWithdraw is going to greater than or equal to zero. Maybe then another nested conditional to add the zero class to turn the background red when it does == 0 and then don't allow anything to happen if it goes negative. Maybe send out an alert saying.."whoops you don't have the money?"